ci/generate-docs.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
```

Comment:
Include some setup instructions in README for getting `hub`, as well as some docs for how to use this script - what's the overall process for docs generation, when should it be run?

Reply:
Currently manually - this belongs somewhere in the release process doc https://github.com/spatialos/gdk-for-unity/wiki/Release-Process